python/plugins/db_manager/db_plugins/gpkg/plugin.py

```diff
@@ -189,7 +191,7 @@ def ogrUri(self):
     def mimeUri(self):
 
         # QGIS has no provider to load Geopackage vectors, let's use OGR
-        return u"vector:ogr:%s:%s" % (self.name, self.ogrUri())
+        return u"vector:ogr:%s:%s" % (self.name, re.sub(":", "\:", self.ogrUri()))
```

Comment:
You don't need `re`, a simple `replace` should be faster

Reply:
ok